src/app/Recordings/Filters/DatetimeFilter.tsx

```diff
@@ -152,17 +152,17 @@ export const DateTimeFilter: React.FunctionComponent<DateTimeFilterProps> = ({ o
         const tz = getTimezone(shortName);
         if (!tz) {
           return {
-            ...old,
             text: value,
             date: undefined,
+            timezone: undefined,
             validation: ValidatedOptions.error,
           };
         }
 
         const extractDatetime = parts.slice(0, parts.length - 1).join(' ');
         const d = dayjs(
           extractDatetime,
-          `${dayjs.Ls[dayjs.locale()].formats.L} ${dayjs.Ls[dayjs.locale()].formats.LTS}`,
+          `${dayjs.localeData().longDateFormat('L')} ${dayjs.localeData().longDateFormat('LTS')}`,
```

Comment:
@andrewazores Back on this topic again, I ran across this and it seems `dayjs` cannot reliably parse localized meridiem. For example, it could not parse `PM` in Korean (throwing invalid) but `AM` is recognized. And formatting/displaying is working perfectly. This is quite common to other locales too (e.g. Bangladesh, ...)

Sample fail :https://runkit.com/63cf2e7d1343950008a5db24/63cf2e7ea6283e00086eeee6

I am think we should restrict to support only English variants since that what we have for Languages as now? Or maybe a more standard input format (i.e. ISO8601)?

Reply:
Looks there are a bunch of reports about this: Looks like https://github.com/iamkun/dayjs/issues?q=is%3Aissue+is%3Aopen+meridiem

Reply:
I think using ISO8601 for the input format is probably the best bet.